### ISSUE TITLE
[WzA] Problem z nadaniem przesyłki 

### ISSUE THREAD

**Issue by jzby-dev** [allegro/allegro-api#10262]
Próbuję nadać przesyłkę do paczkomatu InPost, w dostępnych metodach wysyłki mam jedną formę dostawy tego typu, tj. "Allegro Paczkomaty InPost (xxx)" i to jej próbuję użyć. W odpowiedzi dostaję komunikat:

Błąd nadawania przesyłki.Błąd. (commandId: 6d2f939f-76a8-4618-9e2d-97afcb5176cd) DELIVERY_METHOD_NOT_AVAILABLE :: deliveryMethodId You cannot create a shipment using the Separate Agreement. The method assigned to this order is Allegro Standard. Change the agreement and try again.

Jak rozumiem muszę mieć dla tej przesyłki odrębną umowę na paczkomaty w ramach Allegro Standard, zgadza się?

**Comment by jzby-dev** [comment]
Sprawdziłem, id formy dostawy się zgadzają. Po CommandId (effb3ded-7309-462a-b590-f8d54db9c784) nie da się tego sprawdzić?

**Comment by PrzemyslawLukanowski** [comment]
Tak, teraz udało się znaleźć, dziękuję. Widzę jednak, że metoda dostawy w zamówieniu powiązanym z tym mailem jest inna, jest to 0b9bed2c-0bc1-4e1f-9694-29bb39ebb483 (Allegro One Box, One Kurier)

**Comment by jzby-dev** [comment]
Co więcej pracownik przez stronę wygenerował list przewozowy do paczkomatu dla tej przesyłki.

**Comment by PrzemyslawLukanowski** [comment]
Nie, w momencie kiedy zamówienie otrzymuje status READY_FOR_PROCESSING metoda nie może ulec zmianie. Podejrzewam, że patrzysz na zamówienie 9d05b5b0-9f4d-11ef-87e4-2bf54b2a08cc, które jest od tego samego kupującego, jednak z adresem mailowym, którego użyłeś w command Id effb3ded-7309-462a-b590-f8d54db9c784, powiązane jest inne zamówienie, mianowicie - 4cbe8320-a001-11ef-a10d-919db93b27ee.

**Comment by jzby-dev** [comment]
Ok, to chyba wyjaśnia sytuację, ktoś podczas realizacji zamówień pomieszał formy dostawy w zamówieniach. Dzięki.
